bascule/basculehttp/enforcer.go

```diff
@@ -6,8 +6,15 @@ import (
 	"github.com/Comcast/comcast-bascule/bascule"
 )
 
+// Behavior on not found
+const (
+	Forbid = iota
```

Comment:
`iota` is perfect here.  However, it's best to use a custom type so that code is self-documenting:

```
type NotFoundBehavior int

const (
    Forbid NotFoundBehavior = iota
    Allow
)
```

You can also create a `String()` method for that type so that it shows up as something more interesting than "0" or "1" in printf and logging statements.